# @ref adasis_v2.h 
int64 time_stamp

uint8 type

uint8 cyclic_counter

uint8 path_index

uint16 offset

uint8 position_index

uint16 positioin_age

float32 slope

float32 relative_heading

float32 position_probability

uint8 position_confidence

uint8 current_lane
